shop/src/main/java/ro/msg/learning/shop/service/CustomerService.java

```diff
@@ -0,0 +1,38 @@
+package ro.msg.learning.shop.service;
+
+import org.springframework.stereotype.Service;
+import ro.msg.learning.shop.dto.save.SaveCustomerDto;
+import ro.msg.learning.shop.model.Customer;
+import ro.msg.learning.shop.repository.CustomerRepository;
+
+import javax.transaction.Transactional;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class CustomerService {
+    private final CustomerRepository customers;

```

Comment:
lets give things names that really describe them, such as "customerRepository" for this one. It will be much easier to debug and to read. Your future self (and I) will thank you